language/stdlib/modules/LibraTimestamp.move

```diff
@@ -19,7 +33,19 @@ module LibraTimestamp {
         move_to(association, timer);
     }
 
-    // Update the wall clock time by consensus. Requires VM privilege and will be invoked during block prologue.
+    /// Marks that time has started and genesis has finished. This can only be called from genesis.
+    public fun set_time_has_started(association: &signer) acquires CurrentTimeMicroseconds {
+        assert(Signer::address_of(association) == CoreAddresses::ASSOCIATION_ROOT_ADDRESS(), 1);
+
+        // Current time must have been initialized.
+        assert(exists<CurrentTimeMicroseconds>(
```

Comment:
Nit: this formatting is somewhat hard to read

Reply:
Done